src/app/features/video-list/item/item.component.html

```diff
@@ -1,19 +1,29 @@
-<div class="item">
-  <div class="thumbnail">
-    <img [src]="thumnbnailPath" (click)="playRightPlatform(platform, videoId)" />
+<div class="video_tile_grid" *ngIf="!gridSwitch">
+  <div class="video_tile_grid__header">
+    <div class="video_tile_grid__header__image">
+      <img [src]="thumnbnailPath" (click)="play(video.playLink, video.videoId)" />
+    </div>
+    <div class="video_tile_grid__header__title">
+      <a [routerLink]="['video', videoId]" href="#">{{ title | short }}</a>
+    </div>
   </div>
-  <div class="info">
-    <div class="title">
-      {{ title }}
+  <div class="video_tile_grid__footer">
+    <div class="video_tile_grid__info">{{ likes | thousand }} 👍 {{ viewCount | thousand }} 👀</div>
```

Comment:
a co to za ikona  ? :D lepiej wykorzystaj font icons i dodaj stylizacje

Reply:
Wiem, też mi się nie podoba, chciałem szybko skrócić "wyświetleń".